src/amo/components/SearchResult/index.js

```diff
@@ -1,27 +1,32 @@
+/* @flow */
+/* eslint-disable react/sort-comp */
```

Comment:
Does this rule need to be added to our general lint rules, I notice it being used a couple of times?

Reply:
Heh, maybe. It has bugs triggered by Flow otherwise I suppose it works. I would be fine to remove it completely. I've never found it helpful.